busylight/api/busylight_api.py

```diff
@@ -499,7 +542,8 @@ async def flash_light_impressively(
     rgb_a = parse_color_string(color_a, dim)
     rgb_b = parse_color_string(color_b, dim)
 
-    fli = Effects.for_name("blink")(rgb_a, speed.duty_cycle / 10, off_color=rgb_b)
+    fli = Effects.for_name("blink")(
+        rgb_a, speed.duty_cycle / 10, off_color=rgb_b)
```

Comment:
Not an intentional change.

My editor probably auto-linted this file. Let me know if you'd like me to revert this.

Reply:
Not to worry.